src/MainWindow.h

```diff
@@ -456,6 +464,23 @@ private Q_SLOTS:
     bool isInsideStyleBlock(const QString & styleSheet, const QString & styleBlockStartSearchString,
                             const int currentIndex, bool & error) const;
 
+    bool                    getShowNoteThumbnails() const;
+
+    /**
+     * Get a set with note local uids.
+     */
+    QSet<QString>           getHideNoteThumbnailsFor() const;
+
+    /**
+     * Toggle value of "hide current note thumbnail".
+     */
+    void                    toggleHideNoteThumbnailFor(QString noteLocalUid) const;
+    /**
+     * Toggle value of "show note thumbnails".
+     */
+    void                    toggleShowNoteThumbnails() const;
+
+
```

Comment:
The indentation above looks kinda strange and not similar to how other methods are structured. Probably your IDE did this but you should probably not trust it that much and review the diff for each commit (or at least diff between branches when doing a PR) manually.

Reply:
What I wrote about tabular indentation in coding style was about *member variables*, not about methods :smiley: So please do use tabular indentation for member variables but please don't use it for methods.